content/pages/changelog/index.md

```diff
@@ -13,6 +13,23 @@ For historical changelog entries, please see our
 [2019](page://changelog/2019) and
 [2018 release notes](page://changelog/2018).
 
+## 2020-06-11 / New enterprise events for tasks
+
+Starting today, the [enterprise event stream](g://events/for-enterprise/) will
+begin producing new task and task assignment events and some existing task
+events will return additional fields.
+
+New task and task assignment events:
+
+* Task update
+* Task assignment delete
+
+Existing task and task assignment events with new fields:
+
+* Task assignment create
+* Task assignment update
+* Task create
```

Comment:
Do we have details on these new fields? We should probably add these here.

Reply:
We do. How should that be represented? Sub bullets or in-line with the bullets?

Reply:
Code samples in-line with bullets is not doable in markdown. A better idea might be to use sub-headers (H3) for each of these, and then explain and show the fields. Or, if the fields are the same for all events, to just document the new fields under the list.